spec/controllers/queue_items_controller_spec.rb

```diff
@@ -157,7 +157,7 @@
 
       it "doesn't update the list order if the data is a string" do
         post :update_queue_items, queue_items: [{id: queue_item_one.id, list_order: 3}, {id: queue_item_two.id, list_order: "foobar"}]
-        expect(queue_item_one.reload.list_order).to eq(1)
+        expect(QueueItem.first.list_order).to eq(1)
```

Comment:
I prefer the previous version because it was explicit. There are scenarios where `QueueItem.first` would return an object other than `queue_item` (thereby falsely passing or failing the spec). Using `queue_item.reload` guarantees that you are dealing with the same object.